assets/javascripts/discourse/templates/components/chat-message.hbs

```diff
@@ -151,7 +151,15 @@
 
           {{#if message.error}}
             <div class="chat-send-error">
-              {{message.error}}
+              {{#if (eq message.error "network_error")}}
+                <DButton
+                  class="btn-primary btn-small retry-staged-message-btn"
```

Comment:
Nitpicky, but this button as primary seems large. Maybe the situation calls for it, though. (Could also try with a `btn-flat`.)

Reply:
My feeling is that it's not supposed to show often, so probably ok, but we can iterate on it of that becomes an issue, no objection